MumbleCommunicator.cpp

```diff
@@ -78,6 +78,7 @@ void mumble::MumbleCommunicator::connect(MumbleCommunicatorConfig &config) {
     mumConfig.opusEncoderBitrate = config.opusEncoderBitrate;
     mumConfig.cert_file = config.cert_file;
     mumConfig.privkey_file = config.privkey_file;
+    mumConfig.opusSampleRate = 48000;
```

Comment:
remove it, mumlib is set to 16000 by default.

Reply:
16000 didn't work for me and resulted in all audio received by other mumble clients to be massively downpitched and slowed down (as is to be expected with a sample rate mismatch) while telephony clients straight up didn't receive any audio (I suspect due to the codec cutting out extremely high frequencies) until I increased the samplerate.

Reply:
Ah I see you've changed the sip sampling rate, I thought the sampling rate mismatch was between mumlib and the mumble server but I suppose this means it was between the two communicators.

Reply:
build and test again without it, i made and pushed some fixed for this issue.

Reply:
FYI, i performed all wire audio circuit on mumlib for 16Khz for 10 ms and lower latencies, but by default is set to 20 ms plus 1 size buffer, setting to 10 ms you will need to do some changes on your network nodes for beacons intervals, DTIMs intervals, Fragmentation Threshold, RTS Threshold and QoS settings to make it work.

Reply:
Ah okay, that might explain why I had trouble getting 10ms to work.